lib/ui/widgets/settingsView/about_widget.dart

```diff
@@ -29,7 +29,7 @@ class _AboutWidgetState extends State<AboutWidget> {
                         text: 'Version: ${snapshot.data!['version']}\n'
                             'Model: ${snapshot.data!['model']}\n'
                             'Android Version: ${snapshot.data!['androidVersion']}\n'
-                            'Arch: ${snapshot.data!['arch']}\n',
+                            'Supported Arch: ${snapshot.data!['supportedArch'].join(", ")}\n',
```

Comment:
Where is this used? it might also need the plural support

Reply:
When you long-press on the About list, it will copy the value to the clipboard.

As for plural support, will do!

Reply:
When you long-press on the About list, it will copy the value to the clipboard.

As for plural support, will do!

Reply:
Resolved in 69f325337ada87a616c389e7e07e289939e30bdb